netmaster/master/network.go

```diff
@@ -177,6 +177,12 @@ func CreateNetwork(network intent.ConfigNetwork, stateDriver core.StateDriver, t
 		return nil
 	}
 
+	aci, _ := IsAciConfigured()
```

Comment:
The code change looks good. However, we need to add a system test case to cover this. You could this to TestACIMode and verify "docker network ls" shows only epgs.